src/types/__snapshots__/resolveStatements.spec.ts.snap

```diff
@@ -510,6 +510,16 @@ Line 8, col 12:
 "
 `;
 
+exports[`resolveStatements should fail statements for case-51 1`] = `
+"<unknown>:4:5: Variable already defined : rec
```

Comment:
The error message is a bit confusing here: it should say "function 'rec' blah blah"